AnkiDroid/src/main/java/com/ichi2/anki/AddonsBrowser.java

```diff
@@ -0,0 +1,438 @@
+                        addonModel = new AddonModel(addonName, addonVersion, addonDev, addonAnkiDroidAPI, addonHomepage, addonType);
+                        addonsNames.add(addonModel);
+                    }
+                }
+            }
+
+            addonsListRecyclerView.setAdapter(new AddonsAdapter(addonsNames, this));
+
+            if (addonsNames.isEmpty()) {
+                addonsMessageLayout.setVisibility(View.VISIBLE);
+            } else {
+                addonsMessageLayout.setVisibility(View.GONE);
+            }
+
+        } else {
+            UIUtils.showThemedToast(this, getString(R.string.error_listing_addons), true);
+        }
+        hideProgressBar();
+    }
+
+
+    /*
+      Show info about clicked addons with update and delete options
+     */
+    @Override
+    public void onAddonClick(int position) {
+        AddonModel addonModel = addonsNames.get(position);
+
+        Dialog infoDialog = new Dialog(this);
+        infoDialog.setCanceledOnTouchOutside(true);
+        infoDialog.setContentView(R.layout.addon_info_popup);
+
+        TextView name = infoDialog.findViewById(R.id.popup_addon_name_info);
+        TextView ver = infoDialog.findViewById(R.id.popup_addon_version_info);
+        TextView dev = infoDialog.findViewById(R.id.popup_addon_dev_info);
+        TextView jsApi = infoDialog.findViewById(R.id.popup_js_api_info);
+        TextView homepage = infoDialog.findViewById(R.id.popup_addon_homepage_info);
+        Button buttonDelete = infoDialog.findViewById(R.id.btn_addon_delete);
+        Button buttonUpdate = infoDialog.findViewById(R.id.btn_addon_update);
+
+        name.setText(addonModel.getName());
+        ver.setText(addonModel.getVersion());
+        dev.setText(addonModel.getDeveloper());
+        jsApi.setText(addonModel.getJsApiVersion());
+
+        String link = "<a href='" + addonModel.getHomepage() + "'>" + addonModel.getHomepage() + "</a>";
+        homepage.setClickable(true);
+        homepage.setText(HtmlCompat.fromHtml(link, HtmlCompat.FROM_HTML_MODE_LEGACY));
+
+        buttonDelete.setOnClickListener(v -> {
+
+            AlertDialog.Builder alertBuilder = new AlertDialog.Builder(this);
+            alertBuilder.setTitle(addonModel.getName());
+            alertBuilder.setMessage(getString(R.string.confirm_remove_addon, addonModel.getName()));
+            alertBuilder.setCancelable(true);
+
+            alertBuilder.setPositiveButton(
+                    getString(R.string.dialog_ok),
+                    (dialog, id) -> {
+                        // remove the js addon folder
+                        File dir = new File(addonsHomeDir, addonModel.getName());
+                        deleteDirectory(dir);
+
+                        // remove enabled status
+                        SharedPreferences.Editor editor = preferences.edit();
+                        editor.remove(addonModel.getType() + "_addon:" + addonModel.getName());
+                        editor.apply();
```

Comment:
It'd also be useful to extract this string `addonModel.getType() + "_addon:" + addonModel.getName()` into the AddonModel class

```suggestion
                        preferences.edit().remove(addonModel.getType() + "_addon:" + addonModel.getName()).apply();
```